src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -127,6 +131,13 @@ export class CalcitePickListItem {
    */
   @Event() calciteListItemPropsUpdated: EventEmitter;
 
+  /**
+   * Emitted whenever the the item's value property is modified.
+   * @event calciteListItemPropsUpdated
```

Comment:
The event name doesn't match.

Reply:
Should this be named calciteListItemPropsChange? Events shouldn't be in past tense and we've been using change for events.